clients/drcachesim/common/options.cpp

```diff
@@ -504,6 +504,13 @@ droption_t<bytesize_t>
                  "of being simulated.  This skipping may be slow for large skip values; "
                  "consider -skip_instrs for a faster method of skipping.");
 
+droption_t<bytesize_t> op_L0_warmup_refs(
+    DROPTION_SCOPE_CLIENT, "L0_warmup_refs", 0,
+    "Number of memory references for warmup trace",
+    "Specifies the number of memory references included in the warm up trace."
+    "The warmup references come after the fast forward count and before the "
```

Comment:
I think this needs to explain that this uses an online cache whose size is controlled by other options.

Reply:
I have updated the description which addresses this.